posthog/hogql_queries/insights/funnels/base.py

```diff
@@ -37,6 +37,8 @@
 )
 from posthog.types import EntityNode, ExclusionEntityNode
 
+JOIN_ALGOS = "direct,parallel_hash,hash,full_sorting_merge"
```

Comment:
for memory issues why don't we go for `auto`? performance concerns?

Reply:
when i tested it, it took longer and timed out on metabase.